src/main/resources/common/sitemap-entry.jsp

```diff
@@ -2,15 +2,23 @@
 <%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt" %>
 <%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
 <%@ taglib prefix="sitemap" uri="http://www.jahia.org/sitemap" %>
+<%@ taglib prefix="fn" uri="http://java.sun.com/jsp/jstl/functions" %>
+
+<%--@elvariable id="url" type="org.jahia.services.render.URLGenerator"--%>
 
 <c:set var="renderContext" value="${requestScope['renderContext']}"/>
 <c:set var="urlNode" value="${requestScope['urlNode']}"/>
+<c:set var="canUseVanity" value="${sitemap:urlRewriteEnabled()}"/>
```

Comment:
I'm sorry I'm going to be a pain and ask if we can rename the variable here as well to match the function name